src/puppetlabs/puppetdb/query.clj

```diff
@@ -183,12 +183,9 @@
   `negate-term*`."
```

Comment:
the function remove-all-environments in this file is not doing anything anymore, since version is always v4. Additionally as currently written it'll cause a bug when we move to v5.